package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "soda-dashboard",
-    "version": "1.0.0",
+    "version": "1.4.0",
```

Comment:
@ikabir21 There was new pre-release out today. Can you please update the version to `1.4.1`.
thanks

Reply:
sure